File: wrapUpSnacks/python/main_wrap_up_snacks.py
from wrap_up_snacks import *

print("|-------------------|") 
print("| Armstrong Numbers |") 
print("|-------------------|") 
print() 

for number in range(1, 1001) :
    if is_armstrong(number) :
        print("Armstrong number --> " , number) 
    
print() 
print("|-----------------|") 
print("| Perfect Numbers |") 
print("|-----------------|") 
print() 

for number in range(1, 1001) :
    if is_perfect_number(number) :
        print("Perfect number --> " , number) 
    
print() 
print("|-----------------|") 
print("|   Leap Years    |") 
print("|-----------------|") 
print() 

for year in range(1900, 2026) :
    if is_leap_year(year) :
        print("Leap Year --> " , year) 

print() 
print("|---------------------------|") 
print("|   Number Of Leap Years    |") 
print("|---------------------------|") 
print()

leapYearCount = 0 
for year in range(1900, 2026) :
    if is_leap_year(year) :
        leapYearCount += 1
     
print("The Number of leap year is: " , leapYearCount) 

File: wrapUpSnacks/java/MainWrapUpSnacks.java
public class MainWrapUpSnacks{
    public static void main(String[] args) {
        
        System.out.println("|-------------------|");
        System.out.println("| Armstrong Numbers |");
        System.out.println("|-------------------|");
        System.out.println();
        
        for(int number = 1; number <= 1000; number++){
            if(WrapUpSnacks.isArmstrong(number)){
                System.out.println("Armstrong number --> " + number);
            }
        }
        System.out.println();
        System.out.println("|-----------------|");
        System.out.println("| Perfect Numbers |");
        System.out.println("|-----------------|");
        System.out.println();
        
        for(int number = 1; number <= 1000; number++){
            if(WrapUpSnacks.isPerfectNumber(number)){
                System.out.println("Perfect number --> " + number);
            }
        }
        
        System.out.println();
        System.out.println("|-----------------|");
        System.out.println("|   Leap Years    |");
        System.out.println("|-----------------|");
        System.out.println();
        
        for(int year = 1900; year <= 2025; year++){
            if(WrapUpSnacks.isLeapYear(year)){
                System.out.println("Leap Year --> " + year);
            }
        }
        
        System.out.println();
        System.out.println("|---------------------------|");
        System.out.println("|   Number Of Leap Years    |");
        System.out.println("|---------------------------|");
        System.out.println();
        int leapYearCount = 0;
        for(int year = 1900; year <= 2025; year++){
            if(WrapUpSnacks.isLeapYear(year)){
                leapYearCount++;
            }
        }
        System.out.println("The Number of leap year is: " + leapYearCount);
    }
}
